code/modules/mob/living/carbon/human/life.dm

```diff
@@ -368,20 +368,35 @@ All effects don't start immediately, but rather get worse over time; the rate is
 81-90: Extremely high alcohol content - light brain damage, passing out
 91-100: Dangerously toxic - swift death
 */
-
+#define BALLMER_POINTS 5
 /mob/living/carbon/human/handle_status_effects()
 	..()
+
+
 	if(drunkenness)
 		drunkenness = max(drunkenness - (drunkenness * 0.04), 0)
-
 		if(drunkenness >= 6)
 			if(prob(25))
 				slurring += 2
 			jitteriness = max(jitteriness - 3, 0)
 
 		if(drunkenness >= 11 && slurring < 5)
 			slurring += 1.2
-
+		if(mind && (mind.assigned_role == "Scientist" || mind.assigned_role == "Research Director"))
+			if(SSresearch.science_tech)
+				if(drunkenness >= 12.9 && drunkenness <= 13.8)
+					var/ballmer_percent = 0
+					if(drunkenness == 13.35) // why run math if I dont have to
+						ballmer_percent = 1
+					else
+						ballmer_percent = (-abs(drunkenness - 13.35) / 0.9) + 0.5
+
+					if(ballmer_percent > 1)
```

Comment:
Why?  According to the math, the highest balmer percent possible is 0.5, unless you are exactly at 13.35, there's no way for it to be above 1.

Reply:
Figured I'd keep it as insurance

Reply:
There's really no need